### PR TITLE
Fix: vkloadtests link against MoltenVK.xcframework

### PR DESCRIPTION
**Pull request by atteneder**
…those work as well now.

fix: Recent Vulkan SDKs provide static MoltenVK as `.xcframework` and CMake does not support xcframework, so the static .a library is linked directly as a workaround (see https://gitlab.kitware.com/cmake/cmake/-/issues/21752 for details)

### CODE REVIEW

**Comment by MarkCallow** [comment]
Thank you for this. I am wanting to update to a more recent Vulkan SDK.

File: tests/loadtests/vkloadtests.cmake [hunk 1]
```diff
@@ -26,13 +26,22 @@ if(APPLE)
...
+            # so we manually pick the static library file for iOS arm64 from a subfolder here
+            if( IS_DIRECTORY ${VULKAN_SDK}/MoltenVK.xcframework )
+                set( MOLTEN_VK_FRAMEWORK ${VULKAN_SDK}/MoltenVK.xcframework/ios-arm64/libMoltenVK.a )
+            endif()
+        endif()
+        if( NOT EXISTS ${MOLTEN_VK_FRAMEWORK})
+            message(SEND_ERROR "Could not find MoltenVK framework (at VULKAN_SDK dir '${VULKAN_SDK}')")
         else()
-            message("Found MoltenVK.framwork at ${MOLTEN_VK_FRAMEWORK}")
+            message("Found MoltenVK framwork at ${MOLTEN_VK_FRAMEWORK}")
```
Comment (MarkCallow) [inline tests/loadtests/vkloadtests.cmake:44]: Fix the typo "fram"work. I think the . is correct as `MoltenVK.framework` is the name of the package on disk.

**Review by MarkCallow — CHANGES_REQUESTED** [review]

File: tests/loadtests/vkloadtests.cmake (hunk 1 above)
Comment (atteneder) [reply tests/loadtests/vkloadtests.cmake:44]: Will do!
I deliberately removed the dot to make it more generic (newer versions deliver `MoltenVK.xcframework`)

Comment (MarkCallow) [reply tests/loadtests/vkloadtests.cmake:44]: Okay. Just fix the typo then.

**Review by MarkCallow — APPROVED** [review]

**Pull request merged**